src/ProgressOnderwijsUtils/SchemaReflection/DatabaseDescription.cs

```diff
@@ -72,12 +55,9 @@ public IEnumerable<Table> AllTables
         public Table TableByName(string qualifiedName)
             => tableByQualifiedName.Value.TryGetValue(qualifiedName, out var id) ? id : null;
 
-        [CanBeNull]
-        public Table TableByTempDbName(SqlCommandCreationContext conn, string tempName)
-        {
-            var objectId = SQL($"select object_id({$"tempdb..{tempName}"})").ReadScalar<DbObjectId?>(conn);
-            return objectId == null ? null : TableById(objectId.Value);
-        }
+        [NotNull]
+        public Table ExistingTableByName(string qualifiedName)

```

Comment:
Ik snap dat de migratie makkelijker is om de non-nullable een andere naam te geven, maar het voelt wat logischer om het andersom te doen (ala TryGetTableByName en GetTableByName).